patches/acorn@8.11.3.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/package.json b/package.json
+index 1b8dc76afc3cf5890cc3693c2975577fd3117dd6..9ac3a4d813fda1be476bd896a8f6168b3a459e41 100644
+--- a/package.json
++++ b/package.json
+@@ -46,5 +46,6 @@
+   },
+   "bin": {
+     "acorn": "./bin/acorn"
+-  }
++  },
++  "sideEffects": false
+ }
```

Comment:
Could we use `pnpm.packageExtensions` to add the `sideEffects: false` instead?

Reply:
It seems it doesn't work. I guess it only supports dep related fields.